homeassistant/components/husqvarna_automower/sensor.py

```diff
@@ -298,7 +298,9 @@ class AutomowerSensorEntityDescription(SensorEntityDescription):
         key="next_start_timestamp",
         translation_key="next_start_timestamp",
         device_class=SensorDeviceClass.TIMESTAMP,
-        value_fn=lambda data: dt_util.as_local(data.planner.next_start_datetime),
+        value_fn=lambda data: None
+        if data.planner.next_start_datetime is None
+        else dt_util.as_local(data.planner.next_start_datetime),
```

Comment:
Please avoid multi-line ternary operators and lambdas, as they are hard to follow

Reply:
Best thing would be, I already pass `None` or the `datetime.datetime` in UTC from the automower library. But the problem is, that I can't get the current timezone in the library. I need it, because the `next_start` is in the timezone of the device, which is unknown. If I run `datetime.now().astimezone().tzname()` in the HA dev container I got `UTC`, but with the HA helper `dt_util.DEFAULT_TIME_ZONE` I got `Europe/Berlin`. An In my production system I got with `datetime.now().astimezone().tzname()` `CEST` and with `dt_util.DEFAULT_TIME_ZONE` I got `Europe/Berlin`. Can someone help me?